OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -548,16 +548,16 @@ public class LobbyBooleanOption : LobbyOption
 	{
 		static readonly Dictionary<string, string> BoolValues = new Dictionary<string, string>()
 		{
-			{ true.ToString(), "enabled" },
-			{ false.ToString(), "disabled" }
+			{ true.ToString(), "Enabled" },
+			{ false.ToString(), "Disabled" }
 		};
 
 		public LobbyBooleanOption(string id, string name, string description, bool visible, int displayorder, bool defaultValue, bool locked)
 			: base(id, name, description, visible, displayorder, new ReadOnlyDictionary<string, string>(BoolValues), defaultValue.ToString(), locked) { }
 
 		public override string ValueChangedMessage(string playerName, string newValue)
 		{
-			return playerName + " " + BoolValues[newValue] + " " + Name + ".";
+			return playerName + " " + BoolValues[newValue].ToLowerInvariant() + " " + Name + ".";
```

Comment:
I think these kind of strings should be created using `String.Format` - for readability, for not relying on string transformations (to lowercase) and for translation support (it works here only because in English the verb and the adjective have the same form). There are plenty of examples of this pattern throughout the engine.